test/integration/test_load_study.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
this looks good to me, but would be nice to reorganize the test files a bit at some point. We have two copies of  `study_es_0` I believe (one in frontend, one in here). There's also duplication of commands to load this data, since they exist in frontend localdb as well. I guess we could use this script in the `localdb e2e tests` as well now?

Reply:
@inodb there is no duplication of study_es_0 that I am aware of. E2e-tests of frontend operate on study_es_0 of the backend.

>There's also duplication of commands to load this data, since they exist in frontend localdb e2e tests as well. I guess we could use this script in the localdb e2e tests

Indeed, is one of the things to improve. I save this for when The Hyve has less projects for customers. Feel free to pick up this task if you want to.